pages/background.md

```diff
@@ -1,25 +1,49 @@
+ immigration" - Ernest Rubin
+ 
+Our project will look at immigration into the United States. We will look at the major countries that have come to the United States and use data visualization to analyze trends in relation to major world events and immigration laws. The project will look at immigration laws and world events to see patterns or trends in the data and hope to come to conclusions with it. We will be using data found from credible sources and plugging them into Tablaeu to create the visualizations. 
 
-Put a general discussion of your topic here. Don't get into the historiography or scholarship here. Just introduce to your reader what your topic is. What is the 'problem', the 'question', and/or the 'argument'.
 
-YOU SHOULD BE LIBERAL AND INCLUDE ANY AND ALL LINKS WHEN RELEVANT
 
 ---
 
-## Historical Background
+## Historical and Digital Background
 
-This is where your historiography goes. You can discuss the historiography of specific times and places, but you can also discuss any theoretical literature you think is relevant.
+Immigration to the United States has been a staple to our country’s growth both culturally and economically since the nations birth. Historically, there has been multiple causes for immigration as Quinn states. He states that immigrants look to escape conflict, healthcare, escape poverty "The American Dream", provide opportunities toward children, and jobs. The past decade has put immigration up in political strife where the nation is divided on the current situation. Currently as it stands, Mexico is on top of the debate. There is a lot of push and pull for reasons on why Mexicans seek to immigrate here.  Aguila points out three categories for factors that cause immigration flows that can be used as a model for the rest of the countries in this project as well. These categories are:
+
+* _Demand-Pull_ - This is the job availability in the United States when the wages are a significant difference than Mexico's. 
+* _Supply-Push_ - This is the cause of economic stability in Mexico's economy or others that drive people out of the country.
+* _Networks_ - Friends and Family that spark migration from existing residents of the United States
+
+With these categories this can be the base for looking at trends in data. Aguila also shows arguments proving that even low employment rates in Mexico along with decreasing poverty rates still showed a relatively steady pace of migration. Finally, her arguments with networks also serve as a key to migration as her data shows the growth of established communities creating social ties to employments and aiding new migrations to the same occupation. These occupations grow in sync with the growing wage workers and assist in the United States GDP over time. The data and explanations by Aguila are patterns and trends that can be applied to other population growths such as in European populations in the north east and Asian populations in the west of the United States of America. 
```

Comment:
```suggestion
With these categories this can be the base for looking at trends in data. Aguila also shows arguments proving that even low employment rates in Mexico along with decreasing poverty rates still showed a relatively steady pace of migration. Finally, her arguments with networks also serve as a key to migration as her data shows the growth of established communities creating social ties to employments and aiding new migrations to the same occupation. These occupations grow in sync with the growing wage workers and assist in the United States GDP over time. The data and explanations by Aguila are patterns and trends that can be applied to other population growths such as in European populations in the north east and Asian populations in the west of the United States of America (11).
```